src/content/core/okteto-variables.mdx

```diff
@@ -68,58 +291,21 @@ To delete an existing variable, click on the **Delete** button on the right. You
   />
 </p>
 
-## Configuring dynamic endpoints
-
-You can create dynamic endpoints for external resources within your Okteto environment using `$OKTETO_ENV`. You can define `$variables`, assign them to `OKTETO_ENV`, and make those variables available in other sections (e.g. `deploy`, `destroy`) of the Okteto manifest or to generate dynamic endpoints for external resources.
-
-The example below demonstrates adding variables to `$OKTETO_ENV` in the `deploy` section and then accessing those variables in a sample `destroy` section:
-
-```yaml
-deploy:
-  hashicorp/terraform:1.4
-  commands:
-  - name: Create AWS infrastructure
-    command: |
-      set -e
-      resourceName="${OKTETO_NAMESPACE}-oktacoshop"
-      region=$AWS_REGION
-
-      terraform apply -input=false -var "bucket_name=$resourceName" -var "queue_name=$resourceName" -var "region=$region" -auto-approve
-      s3Dashboard="https://s3.console.aws.amazon.com/s3/buckets/${resourceName}"
-
-      # ... shortened for brevity
-    {
-      echo "$resourceName"
-      echo "$region"
-      echo "OKTETO_EXTERNAL_S3_ENDPOINTS_BUCKET_URL=$s3Dashboard"
-      echo "S3_BUCKET_NAME=$resourceName"
-    } >> "$OKTETO_ENV"
-
-destroy:
-  image: hashicorp/terraform:1.4
-  commands:
-  - name: Delete the AWS infrastructure
-    command: |
-      set -e
-
-      # omitted for brevity
-
-      terraform apply -input=false -var "bucket_name=${S3_BUCKET_NAME}" -var "queue_name=$resourceName" -var "region=$region" -auto-approve --destroy
-```
+## Managed by Okteto
 
-### **Default Environment Variables**
+### Default Environment Variables
```

Comment:
This should be the first ones, to follow the same order than in the intro paragraph

Reply:
moved above 👍